bulk_processing/find_invalid_address_case_id_by_uprn.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import csv
+from pathlib import Path
+
+import requests
+from google.cloud import storage
+from requests import HTTPError
+
+from config import Config
+
+
+def generate_bulk_invalid_address_file(file_to_process):
+    case_id_list = []
+    with open(file_to_process, encoding="utf-8") as open_file_to_process:
+        file_reader = csv.DictReader(open_file_to_process, delimiter=',')
+
+        for line_number, row in enumerate(file_reader, 1):
+            case_id_list.extend(get_case_id_from_case_api(row['UPRN'], line_number))
+
+    address_delta_file = write_invalid_addresses_case_id_file(case_id_list, Path(file_to_process))
+    upload_file_to_bucket(address_delta_file)
+
+
+def get_case_id_from_case_api(uprn, line_number):
+    response = requests.get(f'http://{Config.CASEAPI_HOST}:{Config.CASEAPI_PORT}/cases/uprn/{uprn}')
+    try:
+        response.raise_for_status()
+    except HTTPError as e:
+        if response.status_code == 404:
+            print(f'Error 404: Cannot find the UPRN {uprn} on line {line_number}, Error {e}')
+        else:
+            print(f'Network or Internal Server Error on line {line_number}, Error {e}')
```

Comment:
This would be fine for a couple of failures, if we got many it would be a painful manual process to cut them all out of the terminal logs to send back to address management. Not requesting any changes for this ticket, just noting it.